servers/jax-rs-tests/src/main/java/org/projectnessie/jaxrs/AbstractRestNamespace.java

```diff
@@ -45,7 +45,7 @@
 public abstract class AbstractRestNamespace extends AbstractRestRefLog {
 
   @ParameterizedTest
-  @ValueSource(strings = {"a.b.c", "a.b\u0000c.d", "a.b.c.d"})
+  @ValueSource(strings = {"a.b.c", "a.b%00c.d", "a.b.c.d"})
```

Comment:
but `%00` is the canonical representation of dots inside key components, though.

I think we should have both the new `a.b%00c.d` and the old `a.b\u0000c.d` (for compatibility) test case here.

Reply:
I agree with @dimas-b here. We need one canonical representation imo. We could easily end up in a situation where somebody creates a `Namespace` instance using the URL encoded form and ending up a in situation where `a.b%00c.d` != `a.b\u0000c.d` just because the request didn't go through the param converters